wadas/domain/operation_mode.py

```diff
@@ -109,32 +111,96 @@ def _initialize_cameras(self):
                     self.ftp_thread = FTPsServer.ftps_server.run()
         logger.info("Ready for video stream from Camera(s)...")
 
-    def _detect(self, cur_img):
+    @staticmethod
+    def is_video(media_path):
+        """Method to validate if given file is a valid and supported video format"""
+
+        video_formats = r"\.(avi|mov|mp4|mkv|wmv)$"
```

Comment:
Always precompile regexes if they are statically know and used more than once, so that you can directly use them (e.g.: no re.search anymore).